developers_chamber/utils.py

```diff
@@ -27,3 +27,19 @@ def call_compose_command(command):
             compose_process.wait()
         except KeyboardInterrupt:
             compose_process.wait()
+
+
+def print_bold(message):
```

Comment:
Why not use something Click that already provides? https://click.palletsprojects.com/en/7.x/utils/#ansi-colors

Reply:
uh oh, thanks for the tip :+1: